api/auth.go

```diff
@@ -65,12 +66,13 @@ func (auth *Auth) CreateCustomerJWT(customerId int) ([]byte, error) {
 }
 
 // CreateSessionJWT creates a JWT with customer ID and session ID
-func (auth *Auth) CreateSessionJWT(customerId int, session string) ([]byte, error) {
+func (auth *Auth) CreateSessionJWT(customerId int, session string, elevated bool) ([]byte, error) {
 	t := openid.New()
 	t.Set(jwt.IssuedAtKey, time.Now())
 	t.Set(jwt.ExpirationKey, time.Now().Add(MaxSessionAge))
 	t.Set(CustomerID, customerId)
 	t.Set(SessionID, session)
+	t.Set(Elevated, elevated)
```

Comment:
Whether it's elevated or not can be derived from the JWT itself without an explicit claim right? If so, it's probably better not to store it explicitly in the JWT.

Reply:
Not really, there can be a session, but its not clear for the client if this is obtained by the password, irma or dummy flow.

Reply:
Check, and it's only used by the frontend right? So if `elevated` is `true` but no elevation actually took place the backend won't accept that right? If so, this is fine